Model/FilePayload.php

```diff
@@ -39,6 +38,8 @@
 use MauticPlugin\MauticContactClientBundle\Helper\JSONHelper;
 use MauticPlugin\MauticContactClientBundle\Helper\TokenHelper;
 use MauticPlugin\MauticContactClientBundle\Helper\UtmSourceHelper;
+use Sonata\Exporter\Writer\CsvWriter;
```

Comment:
I assume the failed build happened before these changes?

Reply:
Yes, this just to prevent a deprecation notice.

Reply:
reverted to see if it fixes build issues